public/network_vis_controller.js

```diff
@@ -80,6 +79,15 @@ module.controller('KbnNetworkVisController', function ($scope, $sce, $timeout, P
 
         // variables for agg ids
         let secondBucketAggId, colorNodeAggId, edgeSizeAggId
+        let primaryNodeTermName, secondaryNodeTermName, edgeSizeTermName, nodeSizeTermName
```

Comment:
Could you add a comment here that's just like `// variables for tool tip names` or something?